tests/test_marathon_tools.py

```diff
@@ -241,28 +241,28 @@ def test_read_service_config_and_deployments(self):
 
     def test_load_marathon_config(self):
         expected = {'foo': 'bar'}
+        from_file = {'marathon_config': {'foo': 'bar'}}
         file_mock = mock.MagicMock(spec=file)
         with contextlib.nested(
-            mock.patch('paasta_tools.marathon_tools.open', create=True, return_value=file_mock),
-            mock.patch('json.load', autospec=True, return_value=expected)
+            mock.patch('paasta_tools.utils.open', create=True, return_value=file_mock),
```

Comment:
In my opinion we don't need to have these tests anymore. We can let load_system_paasta_config worry about that